ui/src/app/machines/views/MachineList/MachineList.js

```diff
@@ -134,8 +135,8 @@ const generateRows = (rows, hiddenGroups, setHiddenGroups) =>
           content: <NameColumn showMAC={false} systemId={row.system_id} />
         },
         {
-          content: row.power_state,
-          className: "u-upper-case--first"
+          content: <PowerColumn systemId={row.system_id} />,
+          className: "p-table__col--power p-double-row"
```

Comment:
I don't think we need `p-double-row` here - that would just be for the cells that don't have an icon/checkbox in them. Also `p-table__col--power` isn't doing anything yet, but in the original angular css it was responsible for the widths and breakpoints for the power column. Probs fine to just leave it for now